src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -46,6 +47,8 @@ public partial class DynamoModel : INotifyPropertyChanged, IDisposable, IEngineC
         public static readonly int MAX_TESSELLATION_DIVISIONS_DEFAULT = 128;
 
         #region private members
+
+        private readonly string geometryFactoryPath;
```

Comment:
`geometryFactoryPath` is still here. And the reason it is still here because `EngineController` needs the full path to `LibG.ProtoInterface.dll`. It is believed that we can have this removed when [ProtoInterface split](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6405) is completed.